src/components/ConfigurableResource/ResourceForm.tsx

```diff
@@ -48,6 +50,12 @@ const ResourceForm = ({ loading, connection, resource, jwt, token }: IProps) =>
     return acc
   }, {})
 
+  const targetMap = sortedFormFields.reduce((acc: any, formField) => {
```

Comment:
any could be `Record<string, TypeOfFormField>` maybe need to use type argument of reduce